backend/.eslintrc.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
do we want to delete this file? looks like it's in .gitignore

Reply:
Not entirely sure, I am kind of lost on the linter stuff